Scripts/ultraschall_move_left_to_next_edge.lua

```diff
@@ -0,0 +1,6 @@
+#move cursor left to next edge. If one or more items are selected move cursor to the start of most left item.
+if reaper.CountSelectedMediaItems(0)==0 then
+  reaper.Main_OnCommand(41167,0)
```

Comment:
Habe ich es richtig in Erinnerung, dass @mespotine meinte diese direkten Nummernaufrufe sollten wir nicht mehr machen, sondern nur noch die Auflösungen der eigentlichen Namen?

Reply:
Korrekt, da die Nummern sich ändern können, die Namen aber gleich bleiben auf egal welchen Plattformen.
Es sei denn, die Nummer ist eine von Reaper's eigenen native Actions, da können wir uns drauf verlassen, dass die immer gleich bleiben-

Reply:
Ok - kriegen wir das noch in schöner hin und vor allem hiermit kombiniert:

https://github.com/Ultraschall/ultraschall-portable/pull/75/commits/be05ff4a09abd7d1c21d69102a1d5944dd58c99c

?